subtrack.MAUI/Pages/SubscriptionDetails.razor

```diff
@@ -20,7 +20,7 @@ else
                 <h2>@_subscription.Name</h2>
             </div>
             <div>
-                <button aria-label="Edit" type="button" class="btn btn-outline-info"><i class="bi bi-pencil-square"></i></button>
+                <EditButton Url="@EditUrl"></EditButton>
```

Comment:
Might be a dumb question but do we actually have a EditButton component? I dont see youve created it and afaik it does not exist

Reply:
Yepper tossed it in Shared.Components.

Reply:
It doesnt exist for me in the PR 🤷

Reply:
Sorry about that lol